Corrected an error in the sample code on the page sql-api-nodejs-get-started.md

v-alfara opened the pull request:
Sync with `github` code https://github.com/Azure-Samples/documentdb-node-getting-started/blob/master/app.js#L45

azurecla commented:
Hi __@yijiem__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@yijiem Thank you for this contribution.  I'll have the owner of the article review.

@tamram Would you please review and let me know if I should merge this request?

tamram commented:
This change isn't necessary, as stored access policies can be used to provide SAS parameters for blobs and files. 

However, stored access policies can be created only on containers, tables, queues, and file shares - they can't be created on blobs or files. I think this is why the contributor is offering this change. This text doesn't say that they can be created on blobs or files, but I can clarify the text to make that more obvious.

yijiem commented:
Thank you @tamram for clarification!

tysonn commented:
Closing per Tamra's feedback.

v-alfara committed:
Update sql-api-nodejs-get-started.md

Corrected an error in the sample code. 
Sync with `github` code https://github.com/Azure-Samples/documentdb-node-getting-started/blob/master/app.js#L45
